### PR TITLE
igwejk/continue on repository loop error

### PR DESCRIPTION
**Pull request by igwejk**
- Fix spelling 'Defulat' --> 'Default'
- Isolate exceptions due individual repository entry
  Continue with other repository entries if there are issues on some repositories.

### CODE REVIEW

File: src/utils/enableFeaturesForRepository.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import { enableDependabotFixes } from "./enableDependabotUpdates";
+import { enableIssueCreation } from "./enableIssueCreation";
+import { enableActionsOnRepo } from "./enableActions";
+import { auth as generateAuth, client as octokit } from "./clients";
+import { checkIfCodeQLHasAlreadyRanOnRepo } from "./checkCodeQLEnablement";
+
+export const enableFeaturesForRepository = async ({
+  repository,
+}: {
+  repository: any;
```
[the remark this answers is not in the document]
Comment (igwejk) [reply src/utils/enableFeaturesForRepository.ts:22]: Sure, I've defined the type as follows
```typescript
type RepositoryFeatures = {
  enableDependabot: boolean;
  enableDependabotUpdates: boolean;
  enableSecretScanning: boolean;
  enableCodeScanning: boolean;
  enablePushProtection: boolean;
  enableActions: boolean;
  primaryLanguage: string;
  createIssue: boolean;
  repo: string;
};
```

[the remark this answers is not in the document]
Comment (NickLiffen) [reply src/utils/enableFeaturesForRepository.ts:22]: That works 👍 There is a lot I wish I could do with this project :D

[the remark this answers is not in the document]
Comment (igwejk) [reply src/utils/enableFeaturesForRepository.ts]: I've updated the change with the type definition.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "github.copilot",
     "donjayamanne.githistory",
     "nixon.env-cmd-file-syntax",
-    "mattpocock.ts-error-translator"
+    "mattpocock.ts-error-translator",
   ],
   "postCreateCommand": "yarn install --frozen-lockfile && yarn run build",
-  "remoteUser": "root"
+  "remoteUser": "root",
```
Comment (igwejk) [inline .devcontainer/devcontainer.json:20]: This was introduced by `npm run prettier:write`.

Comment (igwejk) [inline .devcontainer/devcontainer.json:20]: This was introduced by `npm run prettier:write`.

File: tsconfig.json [hunk 3]
```diff
@@ -9,6 +9,6 @@
     "noUnusedParameters": true,
     "removeComments": true,
     "preserveConstEnums": true,
-    "resolveJsonModule": true
-  }
+    "resolveJsonModule": true,
+  },
```
Comment (igwejk) [inline tsconfig.json:13]: This was introduced by `npm run prettier:write`.

**Review by NickLiffen — APPROVED** [review]

**Pull request merged**